.travis.yml

```diff
@@ -1,11 +1,13 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.1.6
-  - 2.2.4
+  - 2.2.10
+  - 2.3.7
+  - 2.4.4
+  - 2.5.3
 env:
-  - RAILS_VERSION=4.0.13
-  - RAILS_VERSION=4.1.15
-  - RAILS_VERSION=4.2.7
-before_script: gem install bundler -v 1.12.5
+  - RAILS_VERSION=5.0.7
```

Comment:
Can we also support and test against rails 4.2?

Reply:
Support for Rails 5.1 would also be nice, so we will not update this twice

Reply:
@andriymosin That's what I wanted too, but discovered there are some incompatibilities in this repo between 5.1 and 4.2 in regards to specs.
My plan was to have this version as a stepping stone for moving between 4 and 5, maintaining compatibility for both.  Then we'd release another version that actually works for all 5 versions but is not compatible with 4. Does that make sense?

Reply:
@PolRod I see, thank you for explaining, I didn't know you faced such problems. Now it looks like the only possible solution. Thank you!